contrib/ibmock/config.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef __CONFIG_H
+#define __CONFIG_H
+
+#define NUM_DEVS 2
+
+#include <infiniband/efadv.h>
+#include <infiniband/verbs.h>
+
+static inline int conf_num_devices(void)
```

Comment:
minor: seems redundant. maybe #define is sufficient?

Reply:
fixed